client/state/automated-transfer/middleware.js

```diff
@@ -1,29 +1,26 @@
 /**
  * Internal dependencies
  */
-import localforage from 'lib/localforage';
 import {
-	THEME_TRANSFER_STATUS_RECEIVE,
-	AUTOMATED_TRANSFER_STATUS_SET,
-	AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE
+	THEME_TRANSFER_INITIATE_FAILURE,
+	THEME_TRANSFER_INITIATE_REQUEST,
+	THEME_TRANSFER_INITIATE_SUCCESS,
```

Comment:
Can you verify that this works with AT initiated with plugins? There's a more redux actions with a difference namespace which is annoying. `AUTOMATED_TRANSFER_STATUS_SET` comes to mind as also triggering a start/end state: https://github.com/Automattic/wp-calypso/blob/d9617edb4d10f8fa3fe74304dda71913768ef846/client/state/automated-transfer/reducer.js#L29